src/lib/hover_thrust_estimator/zero_order_hover_thrust_ekf_test.cpp

```diff
@@ -47,32 +47,37 @@ using namespace matrix;
 class ZeroOrderHoverThrustEkfTest : public ::testing::Test
 {
 public:
+	ZeroOrderHoverThrustEkfTest()
+	{
+		_random_generator.seed(42);
```

Comment:
:slightly_smiling_face:

Reply:
Just FYI (and lessons learned from previous hair-pulling), these still won't be the same across platforms or std libraries... :cry:

Reply:
@jkflying hmm, okay, should I run once the Gaussian noise generator and save the output in a file to be sure that we always use the same data?